stdlib/2and3/ftplib.pyi

```diff
@@ -3,7 +3,23 @@ from _typeshed import SupportsRead, SupportsReadline
 from socket import socket
 from ssl import SSLContext
 from types import TracebackType
-from typing import Any, BinaryIO, Callable, Dict, Iterable, Iterator, List, Optional, Text, TextIO, Tuple, Type, TypeVar, Union
+from typing import (
+    Any,
+    BinaryIO,
+    Callable,
+    Dict,
+    Iterable,
+    Iterator,
+    List,
+    Literal,
```

Comment:
Unfortunately, we have to import `Literal` from `typing_extensions`, as not all version of `typing` have this.